src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -121,6 +126,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $options = array();
         } elseif ($parameter = $input->getOption('parameter')) {
             $options = array('parameter' => $parameter);
+        } elseif ($input->getOption('autoconfigure')) {
+            @unlink($this->getApplication()->getKernel()->getContainer()->getParameter('debug.container.dump'));
```

Comment:
The container dump doesn't contain the information concerning autoconfiguring tags. is it problematic? Is there another way? Should I pass an argument to `getContainerBuilder` to force a compilation ?